WooCommerce/Classes/Model/BetaFeature.swift

```diff
@@ -111,5 +120,13 @@ private extension BetaFeature {
         static let inAppPurchasesManagementDescription = NSLocalizedString(
             "Test out in-app purchases as we get ready to launch",
             comment: "Cell description on beta features screen to enable in-app purchases")
+
+        static let pointOfSaleTitle = NSLocalizedString(
+            "betaFeature.description.pointOfSaleTitle",
```

Comment:
super nit: maybe this can be shortened like

```suggestion
            "betaFeature.pointOfSale.title",
```

Reply:
Agree! Updated https://github.com/woocommerce/woocommerce-ios/pull/12653/commits/e10dc188970698cf339c2c1647b52914a67e45c6